data/schemas/com.m3u.data.database.M3UDatabase/16.json

```diff
@@ -227,7 +227,7 @@
       },
       {
         "tableName": "programmes",
-        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`channel_id` TEXT NOT NULL, `epg_url` TEXT NOT NULL, `start` INTEGER NOT NULL, `end` INTEGER NOT NULL, `title` TEXT NOT NULL, `description` TEXT NOT NULL, `icon` TEXT, `categories` TEXT NOT NULL, `id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL)",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`channel_id` TEXT NOT NULL, `epg_url` TEXT NOT NULL, `start` INTEGER NOT NULL, `end` INTEGER NOT NULL, `title` TEXT NOT NULL, `description` TEXT NOT NULL, `new` INTEGER NOT NULL, `new-tag` INTEGER NOT NULL, `live` INTEGER NOT NULL, `live-tag` INTEGER NOT NULL, `previous-start` TEXT, `icon` TEXT, `categories` TEXT NOT NULL, `id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL)",
```

Comment:
It seems that you forgot upgrades the database again

the old json shouldn’t be changed

https://developer.android.com/training/data-storage/room/migrating-db-versions